dubbo-remoting/dubbo-remoting-netty4/src/main/java/org/apache/dubbo/remoting/transport/netty4/NettyClient.java

```diff
@@ -85,6 +94,12 @@ protected void initChannel(Channel ch) throws Exception {
                         .addLast("encoder", adapter.getEncoder())
                         .addLast("client-idle-handler", new IdleStateHandler(heartbeatInterval, 0, 0, MILLISECONDS))
                         .addLast("handler", nettyClientHandler);
+                String socksProxyHost = System.getProperty(SOCKS_PROXY_HOST);
+                if(socksProxyHost != null) {
+                    Integer socksProxyPort = Integer.getInteger(SOCKS_PROXY_PORT, DEFAULT_SOCKS_PROXY_PORT);
```

Comment:
Should be use System.getProperty(SOCKS_PROXY_PORT) ???

Reply:
replace by ConfigUtils.getProperty